app/uploaders/avatar_uploader.rb

```diff
@@ -0,0 +1,52 @@
+class AvatarUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  include CarrierWave::MiniMagick
+  version :thumb do
+  process :resize_to_fill => [50, 50]
+end
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
+  # storage :fog
```

Comment:
All of the comments and non-used options can be safely removed to reduce line noise and potential confusion.